frontend/libs/studio-components/src/components/StudioSpinner/StudioSpinner.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { HTMLAttributes, forwardRef, useId } from 'react';
+import { Paragraph, Spinner } from '@digdir/design-system-react';
+import type { SpinnerProps } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+
+export type StudioSpinnerProps = {
+  spinnerText?: string;
+} & Pick<SpinnerProps, 'size' | 'variant'> &
+  HTMLAttributes<HTMLDivElement>;
```

Comment:
Not sure, but what do you think gives us the best readability? 🤔 

```suggestion

export type StudioSpinnerProps = {
  spinnerText?: string;
  size: SpinnerProps['size'],
  variant: SpinnerProps['variant']
} & HTMLAttributes<HTMLDivElement>;
```

Reply:
I agree! 😄